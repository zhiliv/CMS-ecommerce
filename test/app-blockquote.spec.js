import { shallowMount } from '@vue/test-utils'
import ADDRESS from '@/components/app/blockquote/blockquote.vue'

describe('app-blockquote.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(ADDRESS, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        footer: true
      },

      slots:{
        default: 'Содержимое'
      }
    })
  })
  it('Содержимое html для app-blockquote ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
