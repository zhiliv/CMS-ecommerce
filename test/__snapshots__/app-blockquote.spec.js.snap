// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-blockquote.js Содержимое html для app-blockquote  1`] = `
<blockquote
  class="simple simple-xl"
  id="indent"
  style="background: white;"
>
  Содержимое
</blockquote>
`;
